src/lib/autoinstall/widgets/storage/disk_page.rb

```diff
@@ -0,0 +1,143 @@
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "cwm/page"
+require "autoinstall/widgets/storage/add_children_button"
+require "autoinstall/widgets/storage/disk_device"
+require "autoinstall/widgets/storage/init_drive"
+require "autoinstall/widgets/storage/disk_usage"
+require "autoinstall/widgets/storage/partition_table"
+
+module Y2Autoinstallation
+  module Widgets
+    module Storage
+      # This page allows to edit usage information for a disk
+      class DiskPage < ::CWM::Page
+        # Constructor
+        #
+        # @param controller [Y2Autoinstallation::StorageController] UI controller
+        # @param section [Y2Storage::AutoinstProfile::DriveSection] Drive section corresponding
+        #   to a disk
+        def initialize(controller, section)
+          textdomain "autoinst"
+          @controller = controller
+          @section = section
+          super()
+          self.widget_id = "disk_page:#{section.object_id}"
+          self.handle_all_events = true
+        end
+
+        # @macro seeAbstractWidget
+        def label
+          if section.device && !section.device.empty?
+            format(_("Disk %{device}"), device: section.device)
+          else
+            format(_("Disk"))
+          end
+        end
+
+        # @macro seeCustomWidget
+        def contents
+          VBox(
+            Left(Heading(_("Disk"))),
+            VBox(
+              Left(disk_device_widget),
```

Comment:
just idea when I see all that left. Is possible to use Left on VBox instead on each element?

Reply:
I tried that, but the widgets inside the VBox are misaligned. See the image below:
![misaligned-widget](https://user-images.githubusercontent.com/15836/79111353-45443e00-7d74-11ea-8073-f3ac3568df42.png).

Reply:
OK, maybe bug in libyui itself? I expect it to be properly aligned or maybe just not so nice API to handle it, that needs to do it for all content